### PR TITLE
Support for docker compose v2.0.0

### PR DESCRIPTION
**Pull request by linweiyuan**
## Standards checklist:

- [x] The PR title is descriptive.
- [ ] The PR doesn't replicate another PR which is already open.
- [ ] I have read the contribution guide and followed all the instructions.
- [ ] The code follows the code style guide detailed in the wiki.
- [ ] The code is mine or it's from somewhere with an MIT-compatible license.
- [ ] The code is efficient, to the best of my ability, and does not waste computer resources.
- [ ] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- [...]

## Other comments:

...

### CODE REVIEW

**Comment by fatash89** [comment]
> ## Standards checklist:
> * [x]  The PR title is descriptive.
> * [ ]  The PR doesn't replicate another PR which is already open.
> * [ ]  I have read the contribution guide and followed all the instructions.
> * [ ]  The code follows the code style guide detailed in the wiki.
> * [ ]  The code is mine or it's from somewhere with an MIT-compatible license.
> * [ ]  The code is efficient, to the best of my ability, and does not waste computer resources.
> * [ ]  The code is stable and I have tested it myself, to the best of my abilities.
> 
> ## Changes:
> * [...]
> 
> ## Other comments:
> ...

File: plugins/docker-compose/README.md [hunk 1]
```diff
@@ -13,20 +13,20 @@ plugins=(... docker-compose)
...
-| dcup      | `docker-compose up`            | Build, (re)create, start, and attach to containers for a service |
-| dcupb     | `docker-compose up --build`    | Same as `dcup`, but build images before starting containers      |
-| dcupd     | `docker-compose up -d`         | Same as `dcup`, but starts as daemon                             |
-| dcdn      | `docker-compose down`          | Stop and remove containers                                       |
-| dcl       | `docker-compose logs`          | Show logs of container                                           |
-| dclf      | `docker-compose logs -f`       | Show logs and follow output                                      |
-| dcpull    | `docker-compose pull`          | Pull image of a service                                          |
-| dcstart   | `docker-compose start`         | Start a container                                                |
-| dck       | `docker-compose kill`          | Kills containers                                                 |
+| dco       | `docker compose`               | Docker-compose main command                                      |
```
Comment (JuniorJPDJ) [inline plugins/docker-compose/README.md:16]:
```suggestion
| dco       | `docker compose`               | Docker compose main command                                      |
```

**Comment by mcornella** [comment]
This will break users that aren't using the old version, for example:
```
~ ❯❯❯ docker --version
Docker version 20.10.8, build 3967b7d
~ ❯❯❯ docker compose
docker: 'compose' is not a docker command.
See 'docker --help'
```

**Comment by Joniator** [comment]
Can we merge this as a new, second plugin (e.g. docker-compose-v2), so users can choose if they want to use the new CLI-version?

Alternatively, @linweiyuan could you publish this as a custom plugin in your own repo?

**Comment by Frederick888** [comment]
Perhaps just add an if-else? If @linweiyuan doesn't want to do this I'm happy to submit one.

**Comment by Joniator** [comment]
The autocomplete-Script may need some touch-ups too.

This may also bleed into the docker autocompletion, as it is now no longer a seperate script/binary, but part of the docker binary

**Comment by Frederick888** [comment]
@Joniator Ah, you were right. Then a second plugin with only the aliases makes more sense.

**Comment by mcornella** [comment]
Don't create a second plugin, there has to be a way to detect which way to call it.

**Comment by mcornella** [comment]
Opened #10248 to support the new version, test it out with `omz pr test 10248`. Thanks!

**Comment by mcornella** [comment]
The fix made it into master. Update with `omz update`.

**Pull request closed without merging**